cime_config/testdefs/testlist_clm.xml

```diff
@@ -11,6 +11,7 @@
         aux_cime_baselines: CESM cime baselines (subset of aux_clm tests)
         hillslope: Experimental test list used for the hillslope option of the model
         rxcropmaturity: Short tests to be run during development related to prescribed crop calendars
+        matrixcn: Tests exercising the matrix-CN capability
```

Comment:
I assume that we prefer the matrixcn tests to still run by default. So, clarification... Does the newly introduced matrixcn test-suite run as part of aux_clm? Or do you envision submitting it separately?

Reply:
This is for separate testing when developing CNmatrix stuff. For example, I'm running it right now to test this PR branch.

Reply:
Got it, I should have noticed that they all have aux_clm also listed.